build/openwisp_base/Dockerfile

```diff
@@ -13,33 +15,32 @@ FROM SYSTEM AS PYTHON
 # 1. Twisted: gcc
 # 2. Postgresql: libpq-dev
 RUN apt update && \
-    apt install --yes --no-install-recommends git gcc libpq-dev && \
+    apt install --yes --no-install-recommends gcc libpq-dev && \
     rm -rf /var/lib/apt/lists/* /root/.cache/pip/* /tmp/*
 
 ENV PYTHONPATH=/install/lib/python3.7/site-packages
 # Python Packages
-RUN pip install --editable git+git://github.com/openwisp/openwisp-controller#egg=openwisp_controller \
+RUN pip install --upgrade https://github.com/openwisp/openwisp-controller/tarball/master \
                 --prefix='/install'
-RUN pip install --editable git+git://github.com/openwisp/openwisp-radius#egg=openwisp_radius \
-                --prefix='/install'
-RUN pip install --editable git+git://github.com/openwisp/openwisp-network-topology#egg=openwisp_network_topology \
+RUN pip install --upgrade https://github.com/openwisp/openwisp-network-topology/tarball/master \
                 --prefix='/install'
 RUN pip install --upgrade https://github.com/openwisp/django-freeradius/tarball/master \
                 --prefix='/install'
+RUN pip install --upgrade https://github.com/openwisp/openwisp-radius/tarball/master \
```

Comment:
`openwisp-radius` is moved below because of an on going issue in `openwisp-radius` that needs to be fixed seperately: https://github.com/openwisp/openwisp-radius/issues/73

This is only a quick fix.